src/Database/Connection.php

```diff
@@ -185,7 +184,7 @@ public function connect()
             $this->_driver->connect();
 
             return true;
-        } catch (Exception $e) {
+        } catch (\Exception $e) {
```

Comment:
It's basically this problem:

https://gist.github.com/cvuorinen/e7a19101d40c9fa0a959
https://3v4l.org/cphKu